src/Main.java

```diff
@@ -1,5 +1,80 @@
+import java.util.Arrays;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public static void main(String[] args)
+    {
+        //
+        //задача 1
+        int[] integer = new int[] {1, 2, 3};
```

Comment:
```suggestion
        int[] integer = new int[3];
        for (int i = 0; i < integer.length; i++) {
            integer[i] = i + 1;
        }
```
здесь вот такой вариант предполагался, т.к. `new int[] {1, 2, 3}` то же самое, что `{1, 2, 3}`